Applications/CLI/ogs.cpp

```diff
@@ -92,9 +92,9 @@ int main(int argc, char *argv[])
 	cmd.add(project_arg);
 	cmd.parse(argc, argv);
 
-	ApplicationsLib::LogogSetup logog_setup;
 	ApplicationsLib::LinearSolverLibrarySetup linear_solver_library_setup(
 	    argc, argv);
+	ApplicationsLib::LogogSetup logog_setup;
```

Comment:
Forget about my comment. I did not understand, why that line was moved. But obviously it is because now the formatter might depend on MPI/PETSC which are initialized in the linear solver lib setup.